main.py

```diff
@@ -1,4 +1,280 @@
-import glob, os
-os.chdir('./res')
-for file in glob.glob("*.fb2"):
-    print(file)
+import glob
+import os
+import shutil
+import xml.etree.ElementTree as ET
+from loguru import logger
+import collections
+import re
+import sqlite3
+from sqlite3 import Error
+import timeit
+
+
+# Class provides functionality to search for fb2 files
+class Fb2Reader:
+
+    @logger.catch()
+    def fb2_file_search(self):
+        os.chdir('./res/input')
+        self.__incorrect_file_removal()
+        fb2_file_list = {file for file in glob.glob("*.fb2")}
```

Comment:
Избыточное использование comprehension.

```suggestion
        fb2_file_list = set(glob.glob("*.fb2"))
```